roottest/root/io/filemerger/CMakeLists.txt

```diff
@@ -335,22 +335,22 @@ if(${compression_default} STREQUAL "zlib")
 
          ROOTTEST_ADD_TEST(simple-lz4-compr-level4
                         PRECMD ${ROOT_hadd_CMD} -f404 hsimple404.root hsimple.root
-                        COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple404.root\",25000,404,517262,15)"
+                        COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple404.root\",25000,404,517292,15)"
                         DEPENDS roottest-root-io-filemerger-hsimple)
          if(CMAKE_SYSTEM_PROCESSOR MATCHES aarch64)
             ROOTTEST_ADD_TEST(simple-lz4-compr-level9
                           PRECMD ${ROOT_hadd_CMD} -f409 hsimple409.root hsimple.root
-                          COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple409.root\",25000,409,516975,30)"
+                          COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple409.root\",25000,409,517004,30)"
                           DEPENDS roottest-root-io-filemerger-hsimple)
          else()
             ROOTTEST_ADD_TEST(simple-lz4-compr-level9
                           PRECMD ${ROOT_hadd_CMD} -f409 hsimple409.root hsimple.root
-                          COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple409.root\",25000,409,517008,15)"
+                          COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple409.root\",25000,409,517020,15)"
```

Comment:
```suggestion
                          COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple409.root\",25000,409,517024,15)"
```

Reply:
Do you understand why file size in test is changing?

Reply:
Not really (stored TStreamerInfo contains maybe the number of functions?), but it happens often with this test. See e.g. https://github.com/root-project/root/pull/15021#issuecomment-2013772175 or https://github.com/root-project/roottest/commit/46d2d618c5f32660ae452fb993e3ad0f2173718a by @pcanal

Reply:
New methods in the class should not affect streamer infos. Small adjustment probably ok here, but I did not understand why it happens. @pcanal ?

Reply:
>  but it happens often with this test.

Still it needs to be understood what is changing ....

> New methods in the class should not affect streamer infos.

Indeed, it should not.

Reply:
> Still it needs to be understood what is changing ....

The last one I investigated turned out that it was cause by the inadvertent loading (i.e. bug) of many unwanted libraries.  I.e. it could be serious ... or it might not :)

Reply:
Resolved: we found out that changing comment of a persistent class member is "dangerous" since it changes the StreamerInfo.

Reply:
It is okay to improve the class member documentation.  This is a valid reason to have to update the expected file size.

Reply:
(I did that on Sergei's request). Question: does changing the streamerinfo imply that the ClassDef version must be increased?

Reply:
I think it is better not to change streamer infos (including comments) without real need.
And class version is not need to be changed if only comments are changing.